src/lib/testing/constraint-system.ts

```diff
@@ -389,18 +389,22 @@ function drawFieldVar(): FieldVar {
   let fieldType = drawFieldType();
   switch (fieldType) {
     case FieldType.Constant: {
-      return FieldVar.constant(17n);
+      return FieldVar.constant(1n);
     }
     case FieldType.Var: {
       return [FieldType.Var, 0];
     }
     case FieldType.Add: {
       let x = drawFieldVar();
       let y = drawFieldVar();
+      // prevent blow-up of constant size
+      if (x[0] === FieldType.Constant && y[0] === FieldType.Constant) return x;
```

Comment:
changes here prevent rare cases where `constraintSystem` tests would be flaky on a very large constant input (for example, if the circuit asserts that the input is less than 256)